homeassistant/components/tedee/coordinator.py

```diff
@@ -38,11 +38,12 @@ class TedeeApiCoordinator(DataUpdateCoordinator[dict[int, TedeeLock]]):
     config_entry: ConfigEntry
     bridge: TedeeBridge
 
-    def __init__(self, hass: HomeAssistant) -> None:
+    def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
```

Comment:
```suggestion
    def __init__(self, hass: HomeAssistant, entry: TedeeConfigEntry) -> None:
```

You will also need to either:
- add an import for `TedeeConfigEntry` (with `if TYPE_CHECKING`)
- or move `TedeeConfigEntry` definition to `coordinator.py`

Reply:
I had if TYPE_CHECKING: (look a commit ago), but then CI was failing because of ruff. Do you know how to get around that?

Reply:
Probably because the `if TYPE_CHECKING` solution also needs to have `from __future__ import annotations`

Reply:
Thanks. decided to move it to the coordinator anyways.